products/spring-framework.md

```diff
@@ -9,39 +9,46 @@ auto:
   # See https://rubular.com/r/XQUdQN2MHdmmCD for reference
     regex: '^v?(?<major>[1-9]\d*)\.(?<minor>0|[1-9]\d*)\.(?<patch>0|[1-9]\d*)(\.RELEASE)?$'
 releases:
+-   releaseCycle: "6.0"
+    eol: 2024-08-31
+    support: 2024-08-31
+    latest: "6.0.0"
+    latestReleaseDate: 2022-11-16
+    releaseDate: 2022-11-16
 -   releaseCycle: "5.3"
-    eol: false
-    support: true
+    lts: true
+    eol: 2024-12-31
+    support: 2024-12-31
     latest: "5.3.23"
     latestReleaseDate: 2022-09-15
     releaseDate: 2020-10-27
 -   releaseCycle: "5.2"
     eol: 2021-12-31
-    support: true
+    support: 2021-12-31
```

Comment:
@marcwrobel looks like there is a problem here because its latest version's date is newer than both support and eol dates
Also why  is that support and eol dates are same for all 5.1 , 5.0 , 4.3 ? is these dates correct ?

Reply:
Yes :
- 5.2 was officially eoled on 2021-12-31 (see https://spring.io/projects/spring-framework#support),
- but an exceptional release was done on 2022-05-11 to fix two vulnerabilities (https://spring.io/blog/2022/05/11/spring-framework-5-3-20-and-5-2-22-available-now).

I was a bit surprised too that 5.1, 5.0 and 4.3 share the same eol dates (2020-12-31) but that is what I saw :
- on https://spring.io/projects/spring-framework#support for 5.1
- on https://spring.io/blog/2019/12/03/spring-framework-maintenance-roadmap-in-2020-including-4-3-eol for 5.0 and 4.3 (quote : _The latter are the last maintenance releases in the 5.0.x and 4.3.x lines. We may do critical patches in case of vulnerabilities but otherwise no further releases are planned in those lines until the final cut-off at the end of 2020_).

Reply:
I also resolved the conflict introduced due to the auto update.